picard/ui/metadatabox.py

```diff
@@ -240,7 +266,13 @@ def contextMenuEvent(self, event):
                 menu.addAction(edit_tag_action)
             removals = []
             useorigs = []
+            item = self.currentItem()
             for tag in tags:
+                if tag in self.lookup_tags().keys():
+                    if (item.column() == 1 or item.column() == 2) and len(tags) == 1 and item.text():
```

Comment:
Unrelated to this PR, but replacing 1 and 2 by something meaningful would be better.